src/server/raft_client.rs

```diff
@@ -800,25 +800,27 @@ where
         let (s, pool_len) = {
             let mut pool = self.pool.lock().unwrap();
             if pool.tombstone_stores.contains(&store_id) {
+                let pool_len = pool.connections.len();
+                drop(pool);
```

Comment:
Is it necessary to drop it explicitly?

Reply:
I drop it earlier to reduce the time to hold the lock.